src/components/themed/ExchangedFlipInput2.tsx

```diff
@@ -205,26 +212,38 @@ const ExchangedFlipInput2Component = React.forwardRef<ExchangedFlipInputRef, Pro
     [convertCurrency, cryptoCurrencyCode, defaultIsoFiat, forceField]
   )
 
+  const pluginInfo = getSpecialCurrencyInfo(pluginId)
+  const showMaxButton = pluginInfo.noMaxSpend !== true && hideMaxButton !== true && onMaxPress != null
```

Comment:
`!hideMaxButton` instead of `hideMaxButton !== true`, since now we have a default value.